<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Olivier Trichet &lt;nive@nivalis.org&gt;</author>
 <class>KNode::IdentityEditionDialog</class>
 <widget class="KDialog" name="KNode::IdentityEditionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>531</height>
   </rect>
  </property>
  <widget class="QWidget" name="page" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>407</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="mIdentityManagementLayout">
      <item>
       <widget class="QLabel" name="mIdentitySelectorLabel">
        <property name="text">
         <string comment="@label:listbox">Identity:</string>
        </property>
        <property name="buddy">
         <cstring>mIdentitySelector</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KComboBox" name="mIdentitySelector">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="mButtonNewIdentity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Create an identity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="mButtonDuplicateIdentity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Duplicate this identity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="mButtonRenameIdentity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Rename this identity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KPushButton" name="mButtonRemoveIdentity">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>28</width>
          <height>28</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Remove this identity</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="mEditLinesLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="mNameLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;Your name as it will appear to others reading your articles.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;John Stuart Masterson III&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string comment="@label:textbox Name of the user in a message">Name:</string>
        </property>
        <property name="buddy">
         <cstring>mNameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="mNameEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;Your name as it will appear to others reading your articles.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;John Stuart Masterson III&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mOrganisationLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;The name of the organization you work for.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;KNode, Inc&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Organization:</string>
        </property>
        <property name="buddy">
         <cstring>mOrganisationEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="mOrganisationEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;The name of the organization you work for.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;KNode, Inc&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mEmailLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;Your email address as it will appear to others reading your articles&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;nospam@please.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Email address:</string>
        </property>
        <property name="buddy">
         <cstring>mEmailEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KLineEdit" name="mEmailEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;Your email address as it will appear to others reading your articles&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;nospam@please.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="mReplytoLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;When someone replies to your article by email, this is the address to which the message will be sent. If you fill in this field, please do so with a real email address.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;john@example.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Reply-to address:</string>
        </property>
        <property name="buddy">
         <cstring>mReplytoEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="mReplytoEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;When someone replies to your article by email, this is the address to which the message will be sent. If you fill in this field, please do so with a real email address.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;john@example.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="mMailcopiestoLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;When someone replies to your article on a group, you can ask to be sent a copy by email. This is the address to which this copy will be sent. If you fill in this field, please do so with a real email address.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;john@example.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Mail-copies-to:</string>
        </property>
        <property name="buddy">
         <cstring>mMailcopiestoEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KLineEdit" name="mMailcopiestoEdit">
        <property name="whatsThis">
         <string>&lt;qt&gt;
  &lt;p&gt;When someone replies to your article on a group, you can ask to be sent a copy by email. This is the address to which this copy will be sent. If you fill in this field, please do so with a real email address.&lt;/p&gt;
  &lt;p&gt;Ex: &lt;b&gt;john@example.com&lt;/b&gt;.&lt;/p&gt;
&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="mSigningKeyRequesterLabel">
        <property name="whatsThis">
         <string>&lt;qt&gt;&lt;p&gt;The OpenPGP key you choose here will be used to sign your articles.&lt;/p&gt;&lt;/qt&gt;</string>
        </property>
        <property name="text">
         <string>Signing key:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="Kleo::SigningKeyRequester" name="mSigningKeyRequester" native="true">
        <property name="whatsThis">
         <string>&lt;qt&gt;&lt;p&gt;The OpenPGP key you choose here will be used to sign your articles.&lt;/p&gt;&lt;/qt&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="KPIMIdentities::SignatureConfigurator" name="mSignatureConfigurator" native="true"/>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KDialog</class>
   <extends>QDialog</extends>
   <header>kdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KPIMIdentities::SignatureConfigurator</class>
   <extends>QWidget</extends>
   <header location="global">KPIMIdentities/SignatureConfigurator</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Kleo::SigningKeyRequester</class>
   <extends>QWidget</extends>
   <header>libkleo/ui/keyrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
